reference/concepts/operators.md

```diff
@@ -2,7 +2,8 @@
 
 Operators are constructs which behave generally like [functions][concept-functions], but which differ syntactically or semantically from usual functions. Common simple examples include [arithmetic][concept-arithmetic] (addition with `+`), [comparison][concept-comparisons] (with `>`), and [logical operations][concept-conditionals] (such as `AND` or `&&`). More involved examples include [assignment][concept-assignment] (usually `=` or `:=`), field access in a record or object (usually `.`), and the scope resolution operator (often `::`). Languages usually define a set of built-in operators, and in some cases allow users to add new meanings to existing operators or even define completely new operators.
 
-[concept-arithmetic]: ./arithmetic.md
-[concept-assignment]: ./assignment.md
-[concept-comparisons]: ./comparisons.md
-[concept-conditionals]: ./conditionals.md
+[concept-arithmetic]: arithmetic.md
```

Comment:
I was a little confused at first, as it looked like there were several links that were fixed.

Reply:
This is the PR I was trying to find where I accidentally removed the `./` from the file paths. I'll change it back.

Reply:
Great!